src/index.js

```diff
@@ -0,0 +1,16 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
```

Comment:
- [ ] In the requirement of the project you are asked to add links. When the user clicks on links, a correct path of the page is displayed. Let's try to fix this issue so that every link in the mobile menu works properly.